integrations/extensions/docs/elasticsearch-install-and-setup/watsonx_discovery_install_and_setup.md

```diff
@@ -0,0 +1,430 @@
+* Learn more about [elser-mappings](https://www.elastic.co/guide/en/elasticsearch/reference/8.11/semantic-search-elser.html#elser-mappings) from the tutorial
+
+### Create an ingest pipeline with an inference processor
+Create an ingest pipeline with an inference processor to use ELSER to infer against the data that will be ingested in the pipeline.
+  ```bash
+  curl -X PUT "${ES_URL}/_ingest/pipeline/elser-v2-test?pretty" -u "${ES_USER}:${ES_PASSWORD}" \
+  -H "Content-Type: application/json" --cacert "${ES_CACERT}" -d'
+  {
+    "processors": [
+      {
+        "inference": {
+          "model_id": ".elser_model_2",
+          "input_output": [
+            {
+              "input_field": "text",
+              "output_field": "text_embedding"
+            }
+          ]
+        }
+      }
+    ]
+  }'
+  ```
+Learn more about [inference-ingest-pipeline](https://www.elastic.co/guide/en/elasticsearch/reference/8.11/semantic-search-elser.html#inference-ingest-pipeline) from the tutorial
+
+### Ingest the data through the inference ingest pipeline
+Create the tokens from the text by reindexing the data through the inference pipeline that uses ELSER as the inference model.
+  ```bash
+  curl -X POST "${ES_URL}/_reindex?wait_for_completion=false&pretty" -u "${ES_USER}:${ES_PASSWORD}" \
+  -H "Content-Type: application/json" --cacert "${ES_CACERT}" -d'
+  {
+    "source": {
+      "index": "wa-docs"
+    },
+    "dest": {
+      "index": "search-wa-docs",
+      "pipeline": "elser-v2-test"
+    }
+  }'
+  ```
+* `wa-docs` is the index you created when uploading the example file to Elasticsearch cluster. It contains the text data
+* `search_wa-docs` is the search index that has ELSER output field
+* `elser-v2-test` is the ingest pipeline with an inference processor using ELSER model
+
+### Semantic search by using the text_expansion query
+To perform semantic search, use the `text_expansion` query, and provide the query text and the ELSER model ID.
+The example below uses the query text "How to set up custom extension?", the `text_embedding` field contains
+the generated ELSER output:
+  ```bash
+  curl -X GET "${ES_URL}/search-wa-docs/_search?pretty" -u "${ES_USER}:${ES_PASSWORD}" \
+  -H "Content-Type: application/json" --cacert "${ES_CACERT}" -d'
+  {
+     "query":{
+        "text_expansion":{
+           "text_embedding":{
+              "model_id":".elser_model_2",
+              "model_text":"how to set up custom extension?"
+           }
+        }
+     }
+  }'
+  ```
+NOTE: Learn more about [text-expansion-query](https://www.elastic.co/guide/en/elasticsearch/reference/8.11/semantic-search-elser.html#text-expansion-query) from the tutorial.
+
+## Step 3: Build a custom extension in watsonx Assistant for Elasticsearch API
+
+### Provision a watsonx Assistant instance in your CloudPak cluster
+From you CloudPak cluster, you need to provision a watsonx Assistant instance and then create an assistant in the new wxA experience.
```

Comment:
Can we replace "wxA" with "watsonx Assistant" here?  I think below it is fine because you define the abbrevation and then use it many times.  Here though it is new and kind of confusing.